<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uart_ass</class>
 <widget class="QMainWindow" name="uart_ass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uart_ass</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>81</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="port_num"/>
     </item>
     <item>
      <widget class="QComboBox" name="baud_rate"/>
     </item>
     <item>
      <widget class="QComboBox" name="data_bit"/>
     </item>
     <item>
      <widget class="QComboBox" name="stop_bit"/>
     </item>
     <item>
      <widget class="QComboBox" name="check_bit"/>
     </item>
     <item>
      <widget class="QComboBox" name="rcv_mod"/>
     </item>
     <item>
      <widget class="QComboBox" name="send_mod"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>81</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>串口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>接收格式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>发送格式</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clear_rcv_area">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清空接收区</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clear_send_area">
       <property name="text">
        <string>清空发送区</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="rcv_area">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="send_area">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_mt">
    <property name="title">
     <string>串口助手_mt</string>
    </property>
   </widget>
   <addaction name="menu_mt"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
